app/back_end/src/data/refactoring.py

```diff
@@ -347,3 +349,107 @@ def find_popmax_in_gnomad(data:pd.DataFrame):
+        row.append(", ".join(proteins) if proteins else "")
+
+        # Condition(s)
+        germline_classification = element.find("ClassifiedRecord/Classifications/GermlineClassification")
+        conditions = [
+            inner.text
+            for inner in germline_classification.findall("ConditionList/TraitSet/Trait/Name/ElementValue[@Type='Preferred']")
+            if inner.text is not None
+        ]
+        row.append("|".join(conditions) if conditions else "")
+
+        # Accession
+        accession = element.attrib.get("Accession")
+        row.append(accession if accession is not None else "")
+
+        # GRCh37Chromosome
+        grch37_sequence_location = element.find("ClassifiedRecord/SimpleAllele/Location/SequenceLocation[@Assembly='GRCh37']")
+        grch37_chromosome = grch37_sequence_location.attrib.get("Chr") if grch37_sequence_location is not None else None
+        row.append(grch37_chromosome if grch37_chromosome is not None else "")
+
+        # GRCh37Location
+        grch37_start = grch37_sequence_location.attrib.get("display_start") if grch37_sequence_location is not None else None
+        grch37_end = grch37_sequence_location.attrib.get("display_stop") if grch37_sequence_location is not None else None
+        row.append(f"{grch37_start} - {grch37_end}" if grch37_start is not None and grch37_end is not None and grch37_start != grch37_end else grch37_start if grch37_start is not None else "")
+
+        # GRCh38Chromosome
+        grch38_sequence_location = element.find("ClassifiedRecord/SimpleAllele/Location/SequenceLocation[@Assembly='GRCh38']")
+        grch38_chromosome = grch38_sequence_location.attrib.get("Chr") if grch38_sequence_location is not None else None
+        row.append(grch38_chromosome if grch38_chromosome is not None else "")
+
+        # GRCh38Location
+        grch38_start = grch38_sequence_location.attrib.get("display_start") if grch38_sequence_location is not None else None
+        grch38_end = grch38_sequence_location.attrib.get("display_stop") if grch38_sequence_location is not None else None
+        row.append(f"{grch38_start} - {grch38_end}" if grch38_start is not None and grch38_end is not None and grch38_start != grch38_end else grch38_start if grch38_start is not None else "")
+
+        # VariationID
+        variation_id = element.attrib.get('VariationID')
+        row.append(variation_id if variation_id is not None else "")
+
+        # AlleleID(s)
+        simple_allele = element.find("ClassifiedRecord/SimpleAllele")
+        allele_id = simple_allele.attrib.get("AlleleID") if simple_allele is not None else None
+        row.append(allele_id if allele_id is not None else "")
+
+        # dbSNP ID
+        xrefs = element.findall('ClassifiedRecord/SimpleAllele/XRefList/XRef')
+        xref = next((inner for inner in xrefs if inner.attrib.get("DB") == "dbSNP"), None)
+        row.append(f"{xref.attrib.get("Type")}{xref.attrib.get("ID")}" if xref is not None else "")
```

Comment:
"Type" needs to be in single quotes ('Type'), otherwise it causes errors

Reply:
Resolved with [MDE/KBE-57 Refactored " and ' chars](https://github.com/mantvydasdeltuva/kath/pull/10/commits/de549dc1c0fd78d071c4e3dd4d8f55becbe5283f)